projects/lookup/src/modules/shared/sky-lookup-string-util-service.ts

```diff
@@ -0,0 +1,9 @@
+export class SkyLookupStringUtilService {
+  /**
+   * Removes diacritical characters from strings and replaces them with their normalized equivalent.
+   * For example: è becomes e, Å becomes A.
+   */
+  static normalizeDiacritics(value: string): string {
```

Comment:
You could simply export this function, instead of naming it as a "service" to avoid confusion with Angular injectable services. This will also keep the size of the bundle smaller since it won't need all the JavaScript class structures.
```
export function normalizeDiacritics() {}
```